Terminal.Gui/Views/ListView.cs

```diff
@@ -211,14 +213,50 @@ public int LeftItem
                 throw new ArgumentException ("value");
             }
 
-            _left = value;
+            ScrollLeftOffset = _leftItem = value;
             SetNeedsDisplay ();
         }
     }
 
     /// <summary>Gets the widest item in the list.</summary>
     public int MaxLength => _source?.Length ?? 0;
 
+    /// <inheritdoc/>
+    public override int ScrollLeftOffset
```

Comment:
Initially they weren't virtual but then I realized that sometimes it resulted in recursive execution when the Parent and the scroll bar are always changing each other's content. This was happening with TextView. So I decided to make them virtual so the user can decide what value should be assumed to avoid this recursion.